engine/src/cubos/engine/tools/palette_editor/plugin.cpp

```diff
@@ -0,0 +1,104 @@
+#include <imgui.h>
+
+#include <cubos/core/data/old/debug_serializer.hpp>
+#include <cubos/core/log.hpp>
+
+#include <cubos/engine/imgui/plugin.hpp>
+#include <cubos/engine/renderer/plugin.hpp>
+#include <cubos/engine/tools/asset_explorer/plugin.hpp>
+#include <cubos/engine/tools/palette_editor/plugin.hpp>
+#include <cubos/engine/voxels/plugin.hpp>
+
+using cubos::core::data::old::Debug;
+using cubos::core::ecs::EventReader;
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+using namespace cubos::engine;
+using tools::AssetSelectedEvent;
```

Comment:
Why this?

Reply:
Honestly idk, what do you guys think @GameDevTecnico/cubos-team? I tend to prefer the groups separated by empty lines option, since `using namespace` is different from `using` statements and the order there matters. Still, clang-format doesn't have an option to enforce this and there already lots of places which don't do so.